shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -629,15 +633,17 @@ public void process(RequestProcessor cp) {
         @Override
         public void writeFields(DataOutput out) throws IOException {
             out.writeUTF(segment);
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
```

Comment:
Same comment about backwards compatibility

Reply:
Yes. We need to bump up the protocol version.